### DISCUSSION TITLE
求一个“金十数据”或者“新浪财经”7×24小时的PC应用。

### DISCUSSION THREAD

**Discussion by tycoontin** [tw93/Pake#343] [Q&A]
如题。

**Comment by Tlntin** [comment]
打包不难的，可以参考该[教程](https://github.com/tw93/Pake/wiki/%E5%9C%A8%E7%BA%BF%E7%BC%96%E8%AF%91%EF%BC%88%E6%99%AE%E9%80%9A%E7%94%A8%E6%88%B7%E4%BD%BF%E7%94%A8%EF%BC%89)
